serverless/src/shared/getCsvProviderUrlFlag.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Determines if a given scheme is a CSV provider URL flag.
+ *
+ * @param {string} scheme - The scheme to check.
+ * @returns {boolean} True if the scheme is a CSV provider URL flag, false otherwise.
+ */
+const getCsvProviderUrlFlag = (scheme) => {
```

Comment:
rename to isCsvProviderUrlFlag.js to denote it returns a boolean

Reply:
Done